src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs

```diff
@@ -269,7 +269,7 @@ public void BuildHasNoWarnings (bool isRelease, bool xamarinForms, bool multidex
 		[TestCase ("", new string [0], false)]
 		[TestCase ("", new string [0], true)]
 		[TestCase ("SuppressTrimAnalysisWarnings=false", new string [] { "IL2055" }, true, 2)]
-		[TestCase ("TrimMode=full", new string [0], false)]
+		[TestCase ("TrimMode=full", new string [] { "IL2055" }, false, 1)]
 		[TestCase ("TrimMode=full", new string [] { "IL2055" }, true, 2)]
```

Comment:
Note that you will get 2x warnings in `Release`, one from the trimmer and one from the Roslyn analyzer. This is "by design", and we shouldn't try to hide them or anything.